openpype/hosts/nuke/plugins/load/load_clip.py

```diff
@@ -145,6 +142,13 @@ def load(self, context, name, namespace, options):
             for k in add_keys:
                 if k == 'version':
                     data_imprint.update({k: context["version"]['name']})
+                elif k == 'colorspace':
+                    colorspace = repre["data"].get(k)
+                    colorspace = colorspace or version_data.get(k)
+                    data_imprint.update({
+                        "db_colorspace": colorspace,
+                        "set_colorspace": set_colorspace
```

Comment:
What if `set_colorspace` or `colorspace` are set to `None`? Shouldn't it store empty string in that case?

Side note: From key `"set_colorspace"`is not obvious if it's verb or noun. I would change it to `"used_colorspace"` if it make sense.

Reply:
Good point, thanks ;)